### DISCUSSION TITLE
Will the new bot buttons be added soon?

### DISCUSSION THREAD

**Discussion by DeadSpark** [discord-jda/JDA#1639] [Questions and Help] [answered]
So discord added this new feature for bots which allows the bots to have interactive buttons looking like this:

![7bb017ce52cfd6575e21c058feb3883b](https://user-images.githubusercontent.com/62199117/120105587-2d499c80-c177-11eb-9392-c15a1d5895c5.png)

More info here: https://discord.com/developers/docs/interactions/message-components#buttons-button-object

I was wondering if this will be added in JDA anytime soon

**Comment by gpluscb** [comment]
The PR is being worked on and it's fairly advanced already, see #1501

**Comment by MinnDevelopment** [comment, marked as answer]
Duplicate of #1633
